src/routing/user/UserTabs.tsx

```diff
@@ -0,0 +1,63 @@
+import { ContentPasteOutlined, ForumOutlined, GroupOutlined, SettingsOutlined } from '@mui/icons-material';
+import { Tabs } from '@mui/material';
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import NavigationTab from '../../components/core/NavigationTab/NavigationTab';
+import NavigationTabs from '../../components/core/NavigationTab/NavigationTabs';
+import { RouterLink } from '../../components/core/RouterLink';
+
+const routes = {
+  profile: '/profile',
+  membership: '/membership',
+  organizations: '/organizations',
+  notifications: '/notifications',
+  root: '/',
+};
+
+export interface UserTabsProps {}
+
+const UserTabs: FC<UserTabsProps> = ({ children }) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <NavigationTabs routes={routes}>
```

Comment:
Can't it provide all the config for the tabs and just traverse it rather than creating each tab manually.

Reply:
It is possible. But no in this version.